### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by szabgab**
You can now enable Travis-CI on this repository.

### CODE REVIEW

File: test/test_chat.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import os
+
+class TestChat:
+    def test_chat(self, tmpdir):
+        out_filename = str(tmpdir.join("abc"))
+        for case in ['One', 'Two']:
+            cmd = 'python wp_parser/wp_chat.py -f test/testChat2.txt -n Username{} > {}'.format(case, out_filename)
```
Comment (nmoya) [inline test/test_chat.py]: testChat2.txt seems to be a depdency

Comment (nmoya) [inline test/test_chat.py]: Ops, it's already commited. 🤐

**Comment by nmoya** [comment]
@szabgab What are the other requirements for travis? Shouldn't we prepare our unit tests better so travis have something to check against?

**Comment by szabgab** [comment]
yep, that file was in the repo already.
Results on Travis in my account: https://travis-ci.org/szabgab/whatsapp-parser

**Comment by szabgab** [comment]
You just need to register on travis and enable this repo. Then as you add more tests, Travis will pick them up on every push.

**Comment by nmoya** [comment]
@szabgab Cool! I see that now. Let's merge this. I plan on improving the tests soon.

**Pull request merged**